<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/new/prefix1/img/Crabs_3.jpg);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Calculate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color:red;
border: 3px solid white;
width:500px;
padding:3px 7px;
</string>
    </property>
    <property name="text">
     <string>Посчитать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Delete_all">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>360</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color:red;
border: 3px solid white;
width:500px;
padding:3px 7px;
</string>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plata">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color:red;
border: 3px solid white;
width:500px;
padding:3px 7px;
</string>
    </property>
    <property name="text">
     <string>Платеж</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Calculate_ST">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>380</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color:red;
border: 3px solid white;
width:500px;
padding:3px 7px;
</string>
    </property>
    <property name="text">
     <string>Посчитать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stavvvka">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>340</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color:red;
border: 3px solid white;
width:500px;
padding:3px 7px;
</string>
    </property>
    <property name="text">
     <string>Ставка</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Calculate_SUM_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>380</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color:red;
border: 3px solid white;
width:500px;
padding:3px 7px;
</string>
    </property>
    <property name="text">
     <string>Посчитать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Summa">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>340</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color:red;
border: 3px solid white;
width:500px;
padding:3px 7px;
</string>
    </property>
    <property name="text">
     <string>Сумма</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Calculate_srok">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>380</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color:red;
border: 3px solid white;
width:500px;
padding:3px 7px;
</string>
    </property>
    <property name="text">
     <string>Посчитать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Srok">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>340</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color:red;
border: 3px solid white;
width:500px;
padding:3px 7px;
</string>
    </property>
    <property name="text">
     <string>Срок</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>30</y>
      <width>723</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QSpinBox" name="Stavka">
       <property name="styleSheet">
        <string notr="true">color:white;
background-color:red;
border: 3px solid white;
width:500px;
padding:3px 7px;</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="Sum">
       <property name="styleSheet">
        <string notr="true">color:white;
background-color:red;
border: 3px solid white;
width:500px;
padding:3px 7px;</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
       <property name="singleStep">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="Months">
       <property name="styleSheet">
        <string notr="true">color:white;
background-color:red;
border: 3px solid white;
width:500px;
padding:3px 7px;</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>480</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_ST">
       <property name="styleSheet">
        <string notr="true">color:white;
background-color:red;
border: 3px solid white;
width:500px;
padding:3px 7px;</string>
       </property>
       <property name="text">
        <string>Годовая ставка(%): </string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_SUM">
       <property name="styleSheet">
        <string notr="true">color:white;
background-color:red;
border: 3px solid white;
width:500px;
padding:3px 7px;</string>
       </property>
       <property name="text">
        <string>Сумма кредита(руб):  </string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_SROK">
       <property name="styleSheet">
        <string notr="true">color:white;
background-color:red;
border: 3px solid white;
width:500px;
padding:3px 7px;</string>
       </property>
       <property name="text">
        <string>На какой срок(мес):</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>401</width>
      <height>38</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_ITOG">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;
background-color:red;
border: 3px solid white;
width:500px;
padding:3px 7px;
</string>
       </property>
       <property name="text">
        <string>Ежемесячный платеж: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="Pay">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;
background-color:red;
border: 3px solid white;
width:500px;
padding:3px 7px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
